<!DOCTYPE html>
<html>
  <head>
    <title>Guy Thampakkul</title>
  
    <!-- fonts -->
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400;500;700&display=swap" rel="stylesheet">
  
    <!-- styles -->
    <link rel="stylesheet" href="styles/general.css">
    <link rel="stylesheet" href="styles/navbar-research.css">
    <link rel="stylesheet" href="styles/profile-bar.css">
    <link rel="stylesheet" href="styles/mains.css">
  </head>
  
  <body>
    
    <!-- nav bar -->
    <nav>
      <div class="navbar">
        <div class="nav-left">
          <div class="name-button">Guy Thampakkul</div>
        </div>
        <div class="nav-right">
          <div class="nav-button-home">Home</div>
          <div class="nav-button-research">Research</div>
          <div class="nav-button-teaching">Teaching</div>
          <div class="nav-button-projects">Projects</div>
        </div>
      </div>
    </nav>

    <div style="padding-top: 70px; height: 10px;"></div>

    <p class="heading">Research Projects</p>

    <main>
      <div class="research-title">
        <i>Interactive Clustering Using Depth Quantile Functions (2022-2023)</i>
      </div>
      <div class="research-main-even" >
        <div class="research-info">
          <p class="research-text">
            Clustering is a challenging problem, especially in high dimensions. There may be multiple correct answers for the same problem. Handling outliers is difficult and evaluating a solution&#39;s correctness is nontrivial. Insight and solutions into these problems become more indecipherable  when black-box algorithms such as neural networks are employed. An interactive and informative method for clustering may alleviate some of these difficulties.
          </p>
          <p class="research-text">
            Calculating Depth Quantile Functions (DQFs) is a method for extracting multiscale geometric features from high dimensional and non-Euclidean data clouds. They map pairs of  data points to real-valued single variable functions on the [0,1] interval using ideas of statistical depth. DQFs have been shown to be effective and interpretable as part in classification, anomaly detection, and now, clustering. The clustering algorithm works as a series of anomaly detection problems determined by the user.
          </p>
        </div>
        <div class="research-image">
          <img src="images/R-DQFs.png" style="max-width:100%;">
        </div>
      </div>

      <div class="divider"></div>

      <div class="research-title">
        <i>The Manifold with Flow Hypothesis (2022-2023)</i>
      </div>
      <div class="research-main-even" >
        <div class="research-image">
          <img src="images/R-FA-Synthetic.png" style="max-width:95%;">
        </div>
        <div class="research-info">
          <p class="research-text">
            A familiar tenet of modern machine learning is the “Manifold Hypothesis”: most high-dimensional data collected in the real world is intrinsically parameterized by a low-dimensional manifold.
          </p>
          <p class="research-text">
            We consider the problem of embedding point cloud data sampled from an underlying manifold with an associated flow or velocity. Such data arises in many contexts where static snapshots of dynamic entities are measured, including in high-throughput biology such as single-cell transcriptomics. Existing embedding techniques either do not utilize velocity information or embed the coordinates and velocities independently, i.e., they either impose velocities on top of an existing point embedding or embed points within a prescribed vector field. The <i>FlowArtist</i> is a neural network that embeds points while jointly learning a vector field around the points. The combination allows <i>FlowArtist</i> to better separate and visualize velocity-informed structures. 
          </p>
        </div>
      </div>

      <div class="research-title">
      </div>

      <div class="research-main-even" >
        <div class="research-info">
          <p class="research-text">
            This project was first considered from the perspective of embedding directed graphs in Euclidean space with a vector field in order to capture directionality information. The model postulates that nodes in a directed graphs are observations from a low-dimensional embedding of a manifold with a vector field. Thus, we look to develop algorithms to convert between directed graph affinities and low-dimensional manifolds with a vector field and extracting information from both ambient and embedding spaces.
          </p>
        </div>
        <div class="research-image">
          <img src="images/R-FA-Schematic.png" style="max-width:100%;">
        </div>
      </div>

      <div class="divider"></div>

      <div class="research-title">
        <i>Bangkok Road Networks as Graphs (2021)</i>
      </div>
      <div class="research-main-even" >
        <div class="research-image" style="background-color: salmon;">
          
        </div>
        <div class="research-info" style="background-color: lightblue;">
          <p class="research-text">
            
          </p>
        </div>
      </div>
      
      <div class="divider"></div>

      <div class="research-title">
        <i>Biofilms Research (2020)</i>
      </div>
      <div class="research-main-even" >
        <div class="research-image">
          <img src="images/R-Biofilms.png" style="max-width: 97%;">
        </div>
        <div class="research-info">
          <p class="research-text">
            Biofilms are structured microbial communities embedded within an extracellular matrix often found on surfaces. Because of their structure, biofilms provide great survival sites for pathogenic bacteria by providing protection. Because of their characteristics, biofilms are often found on man-made surfaces such as pipes, ships, and pools and are difficult to get rid of. This project observes E. Coli Genetics in relation to biofilm formation of wild-type and mutant E. Coli through Dictyostelium phagocytosis with the goal of identifying which genes are responsible for the strength and formation of biofilms.
          </p>
        </div>
      </div>

    </main>

    <!--

    <p class="heading">Preprints</p>

    <p class="heading">Publications</p>

    -->

    <div style="
    margin-top: 100px; 
    border: 1px solid lightgray;
    margin-bottom: 70px;
    border-bottom: 1px solid lightgray;">
    </div>

  </body>
</html>